src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class BulldozerProducer
+        implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        return List.of(new Bulldozer());
```

Comment:
You can create a new instance of the class using the parent fields(super). You have not initialized Object.